.github/workflows/sso-e2e-nightly-windows.yaml

```diff
@@ -0,0 +1,238 @@
+name: Podman Desktop SSO E2E Nightly
+
+on:
+  schedule:
+    - cron:  '0 4 * * *'
+  workflow_dispatch:
+    inputs:
+      fork:
+        default: 'containers'
+        description: 'Podman Desktop repo fork'
+        type: string
+        required: true
+      branch:
+        default: 'main'
+        description: 'Podman Desktop repo branch'
+        type: string
+        required: true
+      ext_repo_options:
+        default: 'REPO=podman-desktop-redhat-account-ext,FORK=redhat-developer,BRANCH=main'
+        description: 'Podman Desktop Extension repo, fork and branch'
+        type: string
+        required: true
+      ext_tests:
+        default: '1'
+        description: 'Run E2E tests from extension'
+        type: string
+        required: true
+      npm_target:
+        default: 'test:e2e'
+        description: 'npm target to run tests'
+        type: string
+        required: true
+      podman_remote_url:
+        default: 'https://github.com/containers/podman/releases/download/v5.1.1/podman-remote-release-windows_amd64.zip'
```

Comment:
seems there is a duplication with the following input

Reply:
I am not sure I can define some constant across the yaml file here. these params are default for a workflow_dispatch run which is "on demand" when user inserts the values. The second batch of params bit below is dedicated for a workflow to be run on nightly base, when inputs are not considered.

Reply:
Only what come to me, is that I could probably improve the workflow to have less lines by using env. on workflow level: 
```
...
inputs:
      ...
      podman_remote_url:
        default: 'https://github.com/containers/podman/releases/download/v5.1.1/podman-remote-release-windows_amd64.zip'

jobs:
  windows:
    name: windows-${{ matrix.windows-version }}-${{ matrix.windows-featurepack }}
    runs-on: ubuntu-latest
    env: 
      URL: ${{ github.event.inputs.podman_remote_url || 'https://some.default.url.org' }}
```

Which would eliminate the number of the lines, but not the duplication.

Reply:
The duplication I'm mentioning is that the input of the workflow defines both the podman version and the podman URL and I think the last one can be computed from the first one so there is no need for the 2

Reply:
Ah, ok. I have created an issue to tackle this: https://github.com/containers/podman-desktop-e2e/issues/194. Although, it cannot be simply computed from url here, you can pass url with version as seen in the PR, or also url without any info about the version which contain an archive with either `5.1.1` or ie. `5.2.0-dev`. So the best would be to integrate the version handling in the image itself and then remove version parameter from the workflows.

Reply:
@jeffmaury So now I have removed the input param for version, it is now counted from extracted archive inside the image's scripts.